stubs/_blueman.pyi

```diff
@@ -0,0 +1,58 @@
+from typing import Any, List, Dict, Union, Callable
+from mypy_extensions import TypedDict
+
+ERR: Dict[int, str]
+RFCOMM_HANGUP_NOW: int
+RFCOMM_RELEASE_ONHUP: int
+RFCOMM_REUSE_DLC: int
+RFCOMM_STATES: List[str]
+RFCOMM_TTY_ATTACHED: int
+
+class RfcommDev(TypedDict):
+    id: int
+    channel: int
+    flags: int
+    state: str
+    src: str
+    dst: str
+
+class HciInfo(TypedDict):
+    acl_mtu: int
+    acl_plts: int
+    bdaddr: str
+    dev_id: int
+    features: List[int]
+    flags: int
+    link_mode: int
+    link_policy: int
+    name: str
+    pkt_type: int
+    sco_mtu: int
+    stat: Dict[str, int]
+    type: int
+
+class BridgeException(Exception):
+    errno: int
+    def __init__(self, errno: int) -> None: ...
+
+class ConnInfoReadError(Exception): ...
+
+class RFCOMMError(Exception): ...
+
+class conn_info:
+    failed: bool
+    def __init__(self, addr: str, hci_name: str) -> None: ...
```

Comment:
Regarding the implementation, `hci_name` is optional as it has a default value.

Reply:
Hmm, ``Optional[X]`` is the same as ``Union[X, None]`` but we do not allow None. But annotation with ``Union[str, str]`` does not look right to me.

Reply:
Ok, I see I did not account for the default value in the annotation which I'll add for these functions.